blazor-wasm/obj/WebApplication1.csproj.nuget.g.props

```diff
@@ -0,0 +1,24 @@
+﻿<?xml version="1.0" encoding="utf-8" standalone="no"?>
```

Comment:
This file is generated during package restore, it should be ignored from version control.